target/linux/ath79/dts/qca9563_tplink_tl-wpa8630p-v2.dtsi

```diff
@@ -0,0 +1,128 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca956x.dtsi"
+
+/ {
+	compatible = "tplink,tl-wpa8630p-v2", "qca,qca9563";
+	model = "TP-Link WPA8630P v2";
```

Comment:
I'd drop these two lines. They only really make sense in a DTS (and would not be evaluated by anything anyway, as they are simply overwritten by the DTS definitions).